techniques/general/G224.html

```diff
@@ -0,0 +1,189 @@
+  </head>
+  <body>
+    <h1>Accounting for meaningful text indentation and Reflow</h1>
+    <section class="meta"><p class="id">ID: G224</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section>
+    <section id="applicability">
+      <h2>When to Use</h2>
+      <p>All technologies that support style switching</p>
+    </section>
+    
+    <section id="description">
+      <h2>Description</h2>
+      <p>There can be instances where indentation of text is considered important to the presentation and understandability of the text content. If the indentations were to be removed, then meaning or functionality could suffer. For instance, the nesting of unordered lists can be harder to notice if indentations are removed. Additionally, maintaining indentations can be important to help understand code patterns, if not even necessary for the code to function.</p>
+      <p>Since maintaining indentation is important for these instances of text, two-dimensional scrolling of these distinct sections of content could be necessary. However, there are still styling modifications that could be made to help ensure the indentation was maintained, but as much text was made visible within the viewport as possible. Making such adjustments could limit, if not mitigate any two-dimensional scrolling from occurring, making the content easier for people to read.</p>
+      </div>
+   </section>
+    
+    <section id="examples">
+      <h2>Examples</h2>
+      <section class="example">
+        <p>A website conveys information in a nested list format. The understandability of the list hierarchy is supported by the indentation of the list content. While it is important for the individual list item text to reflow, the list hierarchy would suffer if flattened so that all content would fit within a 320 CSS pixel wide viewport.</p>  
+        <p>The following example list contains additional nested lists to provide more context for each of the parent list item's text. To maintain the visual hierarchy, the indentation of each list level remains present, but modified once an author defined CSS breakpoint is met. Additional styles are set for different breakpoints to mitigate against the potential for content to extend beyond a 320px wide viewport, while accounting for visual spacing against the borders of the viewport, to attempt to make the content not feel cramped. At the smallest breakpoint, the content of list items receive a minimum width to mitigate against potentially "squished" content lists with many nested levels. At this breakpoint, each nested list level can be horizontally scrolled into view, and once a nested list is visible within the viewport, only vertical scrolling will be necessary to read the content of the nested list's items.</p>
+<pre>
+&lt;div class="example">
+	&lt;ul>
+		&lt;li>
+			&lt;p>Make a list using one of the HTML list elements, or even an ARIA <code>role=list</code> container.&lt;/p>
```

Comment:
There are a bunch of `<code>` elements in this example that don't have their opening `<` escaped, as opposed to the rest of the markup.

I also noticed the examples in this page all use hard tabs, which makes them look inconsistent with all our other pages, but I suspect it's done intentionally for this page?

Reply:
the hard tabs are done on purpose so that the CSS to adjust the width of tabs actually works.
otherwise, the CSS to adjust the width of the tab wouldn't work.

i'll escape the code markup